lutris/runners/runner.py

```diff
@@ -324,26 +335,32 @@ def install_cli(self):
         install the runner provided in prepare_runner_cli()
         """
         if self.name == "wine":
-           print("Please use the Wine flag To install a wine Runner") 
-           exit()
+            print("Please use the Wine flag To install a wine Runner")
+            exit()
         runner_path = os.path.join(settings.RUNNER_DIR, self.name)
+
         if os.path.isdir(runner_path):
             print(self.name + " is already installed!")
-            exit()
         else:
+            from lutris.gui.dialogs import ErrorDialog
             from lutris.gui.dialogs.download import simple_downloader
-            self.install(version=None, downloader=simple_downloader, callback=None)
-            print(self.name + " is now installed :)")
+            try:
+                self.install(version=None, downloader=simple_downloader, callback=None)
+                print(self.name + " is now installed :)")
+            except RunnerInstallationError as ex:
+                ErrorDialog(ex.message)
 
     def uninstall_runner_cli(runner_name):
         """
         uninstall the runner given in application file located in lutris/gui/application.py
         provided using lutris -u <runner>
         """
         Runner.name = runner_name
```

Comment:
```suggestion
        try:
            runner_class = import_runner(runner_name)
            runner = runner_class()
        except InvalidRunner:
            logger.error("Failed to import Runner: %s", runner_name)
            return
```
InvalidRunner must be included in the import from lutris.runners.